frontend/src/pages/Profile.jsx

```diff
@@ -42,15 +48,17 @@ class Profile extends Component {
                         <Table.Body>
                         <Table.Row>
                             <Table.Cell>Name</Table.Cell>
-                            <Table.Cell>Henry Truong</Table.Cell>
+                            <Table.Cell>{this.state.profiledata.firstName} {this.state.profiledata.lastName}</Table.Cell>
                         </Table.Row>
                         <Table.Row>
                             <Table.Cell>Github</Table.Cell>
-                            <Table.Cell><a href="https://github.com/htruong1">https://github.com/htruong1</a></Table.Cell>
+                            <Table.Cell><a href={this.state.profiledata.github}>{this.state.profiledata.github}</a>
```

Comment:
maybe open the link in a new tab instead?

Reply:
kk